src/helpers/clipboard/clipboard_os_state.ts

```diff
@@ -45,10 +45,16 @@ export class ClipboardOsState extends ClipboardCellsAbstractState {
     this.addMissingDimensions(numberOfCols, numberOfRows, activeCol, activeRow);
     for (let i = 0; i < values.length; i++) {
       for (let j = 0; j < values[i].length; j++) {
+        let content: string;
+        if (clipboardOption?.pasteOption === "onlyFormat") {
+          content = "";
```

Comment:
This should be undefined not an empty string, we don't want to clear the cell when pasting only the format. 
Or better we can just do an early return for `onlyFormat`, we don't need to loop on the cells and dispatch UPDATE_CELL  to do nothing on them.